btsieve/tests/test_data/find_transaction_if_blockchain_reorganisation_with_long_chain/generate_test_data.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+source "../lib.sh"
+
+# This directory is created by docker as root
+sudo rm -rf /tmp/bitcoin
```

Comment:
Looks like this can be done in `docker_run`.
Also, `sudo` should not be needed.